libgd/plan.sh

```diff
@@ -0,0 +1,113 @@
+   2004 Greg Roelofs.
+
+   Portions relating to gdttf.c copyright 1999, 2000, 2001, 2002,
+   2003, 2004 John Ellson (ellson@graphviz.org).
+
+   Portions relating to gdft.c copyright 2001, 2002, 2003, 2004 John
+   Ellson (ellson@graphviz.org).
+
+   Portions copyright 2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007
+   Pierre-Alain Joye (pierre@libgd.org).
+
+   Portions relating to JPEG and to color quantization copyright
+   2000, 2001, 2002, 2003, 2004, Doug Becker and copyright (C) 1994,
+   1995, 1996, 1997, 1998, 1999, 2000, 2001, 2002, 2003, 2004 Thomas
+   G. Lane. This software is based in part on the work of the
+   Independent JPEG Group. See the file README-JPEG.TXT for more
+   information.
+
+   Portions relating to GIF compression copyright 1989 by Jef
+   Poskanzer and David Rowley, with modifications for thread safety
+   by Thomas Boutell.
+
+   Portions relating to GIF decompression copyright 1990, 1991, 1993
+   by David Koblas, with modifications for thread safety by Thomas
+   Boutell.
+
+   Portions relating to WBMP copyright 2000, 2001, 2002, 2003, 2004
+   Maurice Szmurlo and Johan Van den Brande.
+
+   Portions relating to GIF animations copyright 2004 Jaakko Hyvätti
+   (jaakko.hyvatti@iki.fi)
+
+Permission has been granted to copy, distribute and modify gd in
+any context without fee, including a commercial application,
+provided that this notice is present in user-accessible supporting
+documentation.
+
+This does not affect your ownership of the derived work itself,
+and the intent is to assure proper credit for the authors of gd,
+not to interfere with your productive use of gd. If you have
+questions, ask. "Derived works" includes all programs that utilize
+the library. Credit must be given in user-accessible
+documentation.
+
+This software is provided "AS IS." The copyright holders disclaim
+all warranties, either express or implied, including but not
+limited to implied warranties of merchantability and fitness for a
+particular purpose, with respect to this code and accompanying
+documentation.
+
+Although their code does not appear in the current release, the
+authors also wish to thank Hutchison Avenue Software Corporation
+for their prior contributions.
+EOF
+)
+pkg_source="https://github.com/libgd/libgd/releases/download/gd-2.2.4/libgd-2.2.4.tar.xz"
+pkg_shasum="137f13a7eb93ce72e32ccd7cebdab6874f8cf7ddf31d3a455a68e016ecd9e4e6"
+pkg_deps=(
+  core/fontconfig
+  core/freetype
+  core/libjpeg-turbo
+  core/libpng
+  core/libtiff
+  core/zlib
+)
+pkg_build_deps=(
+  core/diffutils
+  core/file
+  core/gcc
+  core/make
+  core/pkg-config
+)
+pkg_lib_dirs=(lib)
+pkg_include_dirs=(include)
+pkg_bin_dirs=(bin)
+pkg_pconfig_dirs=(lib/pkgconfig)
+pkg_description="GD is an open source code library for the dynamic creation of images by programmers."
+pkg_upstream_url="https://libgd.github.io"
+
+do_prepare() {
+  hab pkg binlink -d /usr/bin core/file file
```

Comment:
This is going to mutate the outside environment. It may be better to find where `/usr/bin/file` is hard-coded and use `sed` to replace it during `do_prepare`.

Calling `hab` here assumes a build dep on `core/hab` too. If you absolutely need to put something at /usr/bin/file, it would be better to symlink it to `$(pkg_path_for file)/bin/file` manually and make sure it's removed during `do_end`.